sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -2330,4 +2330,8 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
+  test("SPARK-26402: GetStructField with different names are semantically equal") {
```

Comment:
Shall we move this to `org.apache.spark.sql.SQLQuerySuite`?

Reply:
My bad. Two files have  the same name. Moved to the right one. Thanks.